lib/matplotlib/layout_engine.py

```diff
@@ -101,6 +101,28 @@ def execute(self, fig):
         raise NotImplementedError
 
 
+class NullLayoutEnigne(LayoutEngine):
```

Comment:
```suggestion
class NullLayoutEngine(LayoutEngine):
```
(unless it is on purpose...) More locations to change if not.

Reply:
my inability to spell in not intentional!